janusgraph-core/src/main/java/org/janusgraph/diskstorage/keycolumnvalue/scan/StandardScannerExecutor.java

```diff
@@ -324,10 +324,13 @@ public void run() {
             try {
                 while (keyIterator.hasNext()) {
                     StaticBuffer key = keyIterator.next();
-                    RecordIterator<Entry> entries = keyIterator.getEntries();
-                    if (!keyFilter.test(key)) continue;
-                    EntryList entryList = StaticArrayEntryList.ofStaticBuffer(entries, StaticArrayEntry.ENTRY_GETTER);
-                    queue.put(new SliceResult(query, key, entryList));
+                    if (!keyFilter.test(key)) {
+                        continue;
+                    }
+                    try (final RecordIterator<Entry> entries = keyIterator.getEntries()) {
+                        queue.put(new SliceResult(query, key,
```

Comment:
Maybe keep this as two statements (as it was originally) for readability?

Either way, it's two lines, but the original way breaks it up such that it's easier to read, IMHO.

Reply:
set back to two lines and made the entryList final